tests/cra.test.js

```diff
@@ -31,86 +32,83 @@ test.after(async (t) => {
 
 test('homepage', async (t) => {
   const { url } = t.context.server
-  const response = await fetch(`${url}/`).then((res) => res.text())
+  const response = await got(`${url}/`).text()
 
-  t.regex(response, /Web site created using create-react-app/)
+  t.true(response.includes('Web site created using create-react-app'))
```

Comment:
I don't really mind if we use `t.regex` or `string.includes` as long as we are consistent